packages/uploadthing/server.ts

```diff
@@ -1,6 +1,5 @@
 import { createBuilder } from "./src/upload-builder";
 
-export * from "./src/types";
-export * as utapi from "./src/helper-functions";
```

Comment:
I may be blind, but where did this export go?

Reply:
Ah i exported a bunch of stuff when i got the TS2742 errors and must have removed too many when reverting 😓